IBPSA/Fluid/Chillers/Chiller.mo

```diff
@@ -0,0 +1,299 @@
+within IBPSA.Fluid.Chillers;
+model Chiller
```

Comment:
I suggest giving this model a different name. "`Chiller`", to me, would imply that this is the default and representative choice under this folder, but I imagine it is parallel to other model choices. It would therefore be clearer to name this model in a way where it suggests how it is different from other models. For example in `Buildings` library, we have `Buildings.Fluid.Chillers.{AbsorptionIndirectSteam,Carnot_TEva,Carnot_y,ElectricEIR,ElectricReformulatedEIR}`.
I could also see splitting `Chillers` to two subpackages because this model is substantially different from the others, but we should probably leave this thought for later.

Reply:
Thats a valid point, the names are confusing. I would opt for `ModularReversible` or similar. I don't see how they differ from the Carnot_ models, as it's also without a compression cycle model. Or what difference do you refer to?